index.js

```diff
@@ -51,8 +51,8 @@ if (options.cacheTime !== 0) {
 Process query parameters
  */
 
-function isUrl (input) {
-  return input.toLowerCase().startsWith('http')
+function isFile (input) {
+  return !input.toLowerCase().startsWith('http')
```

Comment:
I recommend using the approach of [checking whether file exist](https://stackoverflow.com/questions/4482686/check-synchronously-if-file-directory-exists-in-node-js)
I wonder if this would be better:
```JavaScript
const fs = require('fs')

function fileFound(input) {
  return fs.existsSync(input)
}

options.url = argv.url ? normalizer(arg.url) : normalizer(arg._[0])
options.file = argv.file ? ( isFile(argv.file) ? argv.file : false ) : ( isFile(argv._[0]) ? argv._[0] : false ) ;
```

Reply:
It is always nice to work with designers :-) Yes, it is a better experience to break if a non-existing file is passed in.

Reply:
Thank you :) , I just saw your new change, I think it will work too.
Just include this check, and I'll merge and close.